web/modules/custom/portland/modules/portland_location_picker/src/Plugin/WebformElement/PortlandLocationPicker.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\portland_location_picker\Plugin\WebformElement;
+
+use Drupal\webform\Plugin\WebformElement\WebformCompositeBase;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * Provides a 'portland_location_picker' element.
+ *
+ * @WebformElement(
+ *   id = "portland_location_picker",
+ *   label = @Translation("Portland location picker"),
+ *   description = @Translation("Provides an element for selecting a location by map or address and storing the lat/lon coordiantes."),
+ *   category = @Translation("Composite elements"),
+ *   multiline = TRUE,
+ *   composite = TRUE,
+ *   states_wrapper = TRUE,
+ * )
+ *
+ * @see \Drupal\portland_location_picker\Element\PortlandLocationPicker
+ * @see \Drupal\webform\Plugin\WebformElement\WebformCompositeBase
+ * @see \Drupal\webform\Plugin\WebformElementBase
+ * @see \Drupal\webform\Plugin\WebformElementInterface
+ * @see \Drupal\webform\Annotation\WebformElement
+ */
+class PortlandLocationPicker extends WebformCompositeBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function formatHtmlItemValue(array $element, WebformSubmissionInterface $webform_submission, array $options = []) {
+    return $this->formatTextItemValue($element, $webform_submission, $options);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function formatTextItemValue(array $element, WebformSubmissionInterface $webform_submission, array $options = []) {
+    $value = $this->getValue($element, $webform_submission, $options);
+
+    $lines = [];
+    $lines[] = ($value['first_name'] ? $value['first_name'] : '') .
```

Comment:
Here is another example of the random fields that don't seem to be related to the parent function. 🤔